### ISSUE TITLE
[Feature request] Dark theme support

### ISSUE THREAD

**Issue by SpiritOTHawk** [niedev/RTranslator#53]
Thank you for wanting to share an idea! But before starting, ensure to check if this feature request respects the following requirements:
- [x ] It is written in English (I can translate what you say, but issues written in English are easier to read for the other users).
- [x ] There is not already a similar feature request among the open or closed issues.

**Describe the solution you'd like**
Please add support for dark theme, which will improve the appearance on devices with black theme and reduce battery consumption on OLED screens.

**Comment by niedev** [comment]
Hi, night mode is not a priority for now, but it will surely be added in the future (I use it often too).
